lib/blemulator/example/example_peripheral.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:typed_data';
+
+import 'package:flutter_ble_lib/blemulator/blemulator.dart';
+
+class SensorTag extends SimulatedPeripheral {
+  SensorTag()
+      : super("SensorTag", "4B:99:4C:34:DE:77", Duration(milliseconds: 500), [
+          SimulatedService(
+              "F000AA00-0451-4000-B000-000000000000",
+              true,
+              [
+                SimulatedCharacteristic(
+                    "12:25:00:00:00:00:00:00:00:00:B0:00:40:51:04:01:AA:00:F0",
```

Comment:
what about using named parameters for every argument?

Reply:
Dart supports it only for optional arguments and these should be required. :(

Reply:
There is already @required annotation in Dart, but it just causes lint warnings instead of compilation error.
  void test(int id, {@required int param}) {}